src/context_store/storage/prisma.py

```
@@ -8,228 +8,801 @@
             await self._client.query_raw("SELECT 1 FROM schema_migrations LIMIT 1")
             return
-        except RawQueryError as e:
-            error_str = str(e).lower()
-            if "does not exist" in error_str or "42p01" in error_str:
-                logger.debug("Table schema_migrations not found, applying system migration")
-                await self._apply_migration(file_path)
-                return
-            raise
-        except PrismaError:
-            raise
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)